index.html

```diff
@@ -1,84 +1,111 @@
-		the code is available in <a href="https://github.com/libduck2/tetick">github</a>.<br><br>
-		knock (duck2 at protonmail point com) for any issues of tetick<br>or questions about life and God.
-		</div></div>
-		<div id="wrong" class="overlay"><div class="overchild">
-		<i>- wrong! your scheduler does the wrong thing! this course is not on 8:40-10:40!</i><br><br>
-		some departments hide their course times or just lie to oibs64 and publish their own schedules.(hello BA)<br><br>
-		however tetick is made to embrace weird departments, late night labs or 10:37-12:31 courses. <br>
-		<b>please</b> drop a mail(see "about" on top) about any course data issue.
-		</div></div>
-		<div id="start_time">8:40</div>
-		<div id="end_time">17:30</div>
-		<table>
-		<tr><td><div class="eatd" id="0"></div></td>
-		<td><div class="eatd" id="1"></div></td>
-		<td><div class="eatd" id="2"></div></td>
-		<td><div class="eatd" id="3"></div></td>
-		<td><div class="eatd" id="4"></div></td>
-		</tr></table></div>
-		<div class="form">
-		day(1-5) <input class="short" id="d">, between <input class="short" id="s"> and <input class="short" id="e">
-		<button id="dontfill"> don't fill </button>
-		<div class="flashlight">
-		<button id="prevb">prev</button>
-		<button id="fl">flashlight</button>
-		<button id="nextb">next</button>
-		</div>
-		<div id="fdate">course data is from </div>
-		</div>
-	</div>
-	<div class="row">
-		<div class="options">
-			<div>dept, like fle or me: <input id="dept" class="short"></div>
-			<div>semester: <input id="semester" class="short"></div>
-			<div><button id="add-musts">add must courses</button></div>
-			<div>
-				<input type="checkbox" id="sncheck"> check for my surname, 2 letters: <input id="surname" class="short"> <br>
-			</div>
-			<div><input type="checkbox" id="nodeptcheck"> don't check for dept</div>
-			<div><input type="checkbox" id="allphantom"> all courses phantom </div>
-			<div>
-				course: <input class="awesomplete" onclick="this.value='';" id="course-list" data-maxitems="5">
-				<button id="add"> add </button>
-			</div>
-			<button id="make"> make schedule </button>
-		</div>
-		<div class="courses">
-			<div style="text-align: center; margin: 0.8em;"> current courses </div>
-		</div>
-	</div>
-</div>
+  <!-- Basic Page Needs
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta charset="utf-8">
+  <title>Tetick</title>
+  <meta name="description" content="">
+  <meta name="author" content="">
 
-<div class="footer">
-	<hr>
-	arrow keys also work.
-</div>
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-<script src="awesomplete.js"></script>
-<script src="data.js"></script>
-<script src="main.js"></script>
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```

Comment:
let's keep the HTML webfont-free please. system fonts are enough for most things, they download 0kb and they don't jitter or show blank text while the actual font is being downloaded.